website/docs/internals/json-format.html.md

```diff
@@ -16,7 +16,15 @@ Since the format of plan files isn't suited for use with external tools (and lik
 
 Use `terraform show -json <FILE>` to generate a JSON representation of a plan or state file. See [the `terraform show` documentation](/docs/cli/commands/show.html) for more details.
 
--> **Note:** The output includes a `format_version` key, which currently has major version zero to indicate that the format is experimental and subject to change. A future version will assign a non-zero major version and make stronger promises about compatibility. We do not anticipate any significant breaking changes to the format before its first major version, however.
+The output includes a `format_version` key, which as of Terraform 1.1.0 has
+value `"1.0"`. The semantics of this version are:
+
+- Backwards compatible changes or additions will be made along with a minor
+  version increment, e.g. `"1.1"`. Consumers of this format should ignore any
+  unknown keys to remain forwards compatible.
+- Any backwards incompatible changes will result in a major version increment,
+  e.g. `"2.0"`. Consumers of this format should reject unknown major format
+  versions.
```

Comment:
We typically try to avoid using the passive voice in our docs, but I know there's some tension in that rule when we're writing "specification-like" documentation, because technical specifications often _do_ use the passive voice and thus we're accustomed to sentences structured like this in that context.

I did take an attempt at active voice alternatives of your two list items here, though I'm presenting them only for consideration and not asserting that they are necessarily better, because I remain kinda ambivalent about whether this is an appropriate writing style for specification-like content:

> * We will increment the minor version, e.g. `"1.1"`, for backward-compatible changes or additions. Ignore any object properties with unrecognized names to remain forward-compatible with future minor versions.
> * We will increment the major version, e.g. `"2.0"`, for changes that are not backward-compatible. Reject any input which reports an unsupported major version.

(while editing this I also swapped "unknown keys" for "object properties with unrecognized names", because "properties" is the term commonly used in JavaScript. Sadly the JSON RFC just refers to these as "name/value pairs", which I always find clumsy to use in prose, so I typically write "property names" in an attempt to meld those two sets of terminology together. Again, I'm just presenting this for consideration, not asserting that it's the "right answer".)

Reply:
+1 on the active voice! :-) Thank you, Martin. @alisdair I'd love to see a revised version of this with the active voice. Then I can take a final pass for flow, etc.